packages/react-vapor/src/components/drop/examples/DropExamples.tsx

```diff
@@ -15,16 +14,10 @@ import {Drop} from '../Drop';
 const modalId: string = 'ModalIDDrop';
 const modalId2: string = 'ModalIDDrop2';
 
-const mapDispatchToProps = (dispatch: (action: IReduxAction<IModalActionPayload>) => void): IModalExamplesProps => ({
-    openModal: (id: string) => dispatch(openModal(id)),
-});
-
-@ReduxConnect(null, mapDispatchToProps)
 export class DropExamples extends React.PureComponent<any> {
     openModal(id: string) {
-        this.props.openModal(id);
+        ReactVaporStore.dispatch(openModal(id));

```

Comment:
I feel like the `connect` way is closer to reality

Reply:
you can modify this file to use the new syntax with DropExamplesDisconnected and use connect below after :)

Reply:
:+1:  :heart: realism